.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": false,
+  "quoteProps": "as-needed",
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "bracketSameLine": false,
+  "arrowParens": "always"
+}
```

Comment:
Bit of a hidden project convention is having an empty last line, could we add that to the new files in this PR?

Reply:
I've opened https://github.com/rubygems/rubygems.org/pull/5930 to help us with consistency, especially from agents.